src/Factory.php

```diff
@@ -141,13 +141,25 @@ public function target($name)
             if (preg_match('#^https?://#', $file)) {
                 $files[] = new Remote($file);
             } else {
-                $path = $scanner->find($file);
-                if ($path === false) {
-                    throw new RuntimeException("Could not locate $file for $name in any configured path.");
+                if (preg_match('/(.*)(\/\*.*?)$/', $file, $matches)) {
```

Comment:
You're totally right, I probably am remembering how glob/minimatch works in nodejs. I'm fine with keeping a simple glob for now. We can always include recursive glob later on.

Keeping the supported glob features consistent between `files[]` and `paths[]` options is also nice. This enables simple glob in both, and in the future both can be updated for recursive glob.